### PR TITLE
Document some fields in the CTFE config proto that didn't have any.

### PR DESCRIPTION
**Pull request by Martin2112**
Clarifying wordiology.

### CODE REVIEW

File: trillian/ctfe/configpb/config.proto [hunk 1]
```diff
@@ -50,7 +50,13 @@ message LogConfig {
   // The public key is included for the convenience of test tools (and obviously
   // should match the private key above); it is not used by the CT personality.
   keyspb.PublicKey public_key = 5;
+  // If reject_expired is true then the certificate validity period will be
+  // checked during the validation of submissions. This will cause expired
+  // certificates to be rejected.
   bool reject_expired = 6;
+  // If set ext_key_usages will restrict the set of such usages that the
```
Comment (AlCutter) [inline trillian/ctfe/configpb/config.proto]: "If set, "

File: trillian/ctfe/configpb/config.proto [hunk 2]
```diff
@@ -50,7 +50,13 @@ message LogConfig {
   // The public key is included for the convenience of test tools (and obviously
   // should match the private key above); it is not used by the CT personality.
   keyspb.PublicKey public_key = 5;
+  // If reject_expired is true then the certificate validity period will be
+  // checked during the validation of submissions. This will cause expired
```
Comment (AlCutter) [inline trillian/ctfe/configpb/config.proto]: "checked against the current time during..." just to be clearer?

**Comment by Martin2112** [comment]
I'll do these in a follow up.

**Comment by Martin2112** [comment]
Ah OK it was the other PR I merged. So many things at once ...

File: trillian/ctfe/configpb/config.proto (hunk 1 above)
Comment (Martin2112) [reply trillian/ctfe/configpb/config.proto]: Done.

File: trillian/ctfe/configpb/config.proto (hunk 2 above)
Comment (Martin2112) [reply trillian/ctfe/configpb/config.proto]: Done.

**Pull request merged**